src/Render/Button.php

```diff
@@ -0,0 +1,17 @@
+<?php declare(strict_types=1);
+
+namespace Tribe\Weglot\Render;
+
+class Button {
```

Comment:
There's no reason to use static methods which make testing the classes using them extremely difficult. 

Let's do this instead:

1. Remove the `static` from `get_button()`, developers can inject this using PHP-DI where they need it.
2. Make the markup a public constant, so developers can use that if they really want, e.g. `public const BUTTON_MARKUP = ''<div id="weglot_here"></div>'';`, and make the `get_button()` method return that, but allow it be filtered by a WP filter, say  `tribe/weglot/button_markup`. just in case they need to change/modify it.

Reply:
1. - Got it, and absolutely agree.
2. - Please notice that this is not a button itself, but a needle which is being replaced with a button by weglot.
Also weglot has its own filters for that - 
`public/wp-content/plugins/weglot/src/services/class-generate-switcher-service-weglot.php:51`
So for second I guess I am going to refactor it into a constant and thats it.